cmd/veto/update_runtime.go

```
@@ -0,0 +1,306 @@
+func installUpdate(ctx context.Context, version string, deps updateInstallDeps) error {
+	if _, err := parseStableVersion(version); err != nil {
+		return err
+	}
+	executablePath, err := deps.doctor.executable()
+	if err != nil || executablePath == "" {
+		return errors.New("cannot resolve the running veto executable")
+	}
+	if homebrewManagedPath(executablePath) {
+		brew, err := deps.lookPath("brew")
+		if err != nil {
+			return errors.New("Homebrew installation detected, but brew is not on PATH")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)